collections/made-in-india/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 items:
+ - appsmithorg/appsmith
```

Comment:
Thanks for your contribution! As per contribution guidelines, please move this to the end of the list.

Reply:
Resolved.